__tests__/app.test.js

```diff
@@ -64,5 +64,7 @@ describe("GET /api/articles/:article_id", () => {
         );
       });
   });
+  it("should return with a 404 - not found error if an invalid id is given", () => {
```

Comment:
This is only passing because it is defaulting to a 404, we aren't actually handling the error. There are details on how to handle this in the error-handling notes. You will also need to test what message you expect to come back on this, to ensure it's not getting tangled up with any other 404 errors that you might be handling (e.g, when the user tries to access an endpoint that doesn't exist - such as /api/not-an-endpoint - which you should also add an error-handler in for, please!).